js/src/wp-seo-network-admin.js

```diff
@@ -0,0 +1,96 @@
+/* global wpseoNetworkAdminGlobalL10n, ajaxurl */
+
+import a11ySpeak from "a11y-speak";
+
+( function( $ ) {
+	"use strict";
+
+	/**
+	 * Displays given settings errors.
+	 *
+	 * @param {Object} settingsErrors The list of settings error objects.
+	 *
+	 * @returns {void}
+	 */
+	function displaySettingsErrors( settingsErrors ) {
```

Comment:
minor: this is used for both success and error messages, I'd consider to rename it

Reply:
I'd say we keep the name that way for parity with core. While not great there, the term "settings error" is used for all kinds of settings feedback messages, so the name here should indicate what it is responsible for.